smriprep/cli/utils.py

```diff
@@ -27,16 +29,29 @@ def _template(inlist):
 
 def output_space(value):
     """Parse one element of ``--output-spaces``."""
+    global EXCEPTIONS
     tpl_args = value.split(':')
     template = tpl_args[0]
     spec = {}
     for modifier in tpl_args[1:]:
         mitems = modifier.split('-', 1)
         spec[mitems[0]] = len(mitems) == 1 or mitems[1]
 
+    if EXCEPTIONS and template in EXCEPTIONS:
+        return template, None
+
     if template not in _TF_TEMPLATES:
         raise ValueError("""\
 Template identifier "{}" not found. Please, make sure TemplateFlow is \
 correctly installed and contains the given template identifiers.""".format(template))
 
     return template, spec
+
+
+def set_nonstandard_spaces(inlist):
```

Comment:
This function doesn't seem to be called anywhere.

Reply:
https://github.com/adelavega/fmriprep/commit/c7cddba027946cc8e9c1d524a98b2b855a032adc#diff-b93ab0122f4b2ac6738b896651cac6ccR53

Reply:
Oh, I see...

Is there another way to do this? It feels very messy to fiddle with global state. What if we just passed a list of valid nonstandard spaces to `output_space`?

Reply:
How would you pass it to `ParseTemplates`?

Reply:
IDK... I'm not really aware of all this machinery. I'll look tomorrow to see if I can figure out what we're doing with CLI args these days and if there's something cleaner. If not, I guess do what you have to.

Reply:
For sure I can rework the action class... but I honestly took the lazy way for this.

Reply:
@effigies, more acceptable now?